app/views/users/show.html.haml

```diff
@@ -0,0 +1,42 @@
+%section.user.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %article
+        %header
+          %h2 
+            #{@user.first_name}'s video collections (#{@user.queue_items.count}) 
+          =link_to "Follow", user_friendships_path(current_user, leader_id: @user.id), class: "button btn btn-default", method: "post" if current_user.can_follow?(@user)
+        %table.table
+          %thead
+            %tr
+              %th(width="30%") Video Title
+              %th(width="15%") Genre
+              %th(width="15%") Rating
+          %tbody
+            - @user.queue_items.each do |queue_item|
+              %tr
+                %td
+                  =link_to queue_item.video.title, video_path(queue_item.video)
+                %td
+                  =link_to queue_item.video.category.name, category_path(queue_item.video.category)
```

Comment:
`category_path(queue_item.video.category)` should be just `queue_item.video.category`